src/frontend-samples/snow-decoration-sample/README.md

```diff
@@ -0,0 +1,28 @@
+# Particle Effect - Snow & Rain
+
+Copyright © Bentley Systems, Incorporated. All rights reserved.
+
+This sample shows how to create particle effects using snow and rain as examples.
+
+## Purpose
+
+The purpose of this sample is to demonstrate the following:
+
+* Creating a decorator and rendering it in the active view.
+* Working with particle effects using the [ParticleCollectionBuilder](https://www.itwinjs.org/reference/imodeljs-frontend/rendering/particlecollectionbuilder) API.
+
+## Description
+
+This sample provides an example of a particle effect in view space (as opposed to world space).  The sample provides features to configure a decorator emitting snow particles.  The decorator is creating using predefined props defined in [SnowDecorationApp.ts](./SnowDecorationApp.tsx).  Select which effect is used with the drop down menu.
```

Comment:
...emitting snow and rain...

The decorator is *created* using redefined props